Makefile

```diff
@@ -45,6 +45,12 @@ LDLIBS=\
 
 endif
 
+ifeq ($(OS),linux)
+ifdef JEMALLOC
+LDLIBS+=-ljemalloc
```

Comment:
I am sorry, but I fail to see how it is better than `make LDLIBS=-ljemalloc`. If someone need to build something very specific, he is free to do so via parameters. Does current Makefile prevent this?

We do not want to support and maintain all possible combinations of switches in the Makefile.